src/libcore/iter.rs

```diff
@@ -2448,6 +2459,8 @@ impl<I> Fuse<I> {
     /// previously returned `None`.
     #[inline]
     #[unstable(feature = "iter_reset_fuse", reason = "seems marginal")]
+    #[deprecated(since = "1.3.0",
+                 reason = "unusual for adaptors to have one-off methods")]
```

Comment:
Yeah, I agree: the problem here isn't the one-off-ness per se, but rather that this method doesn't seem terribly useful in practice.